### PR TITLE
Add Calculator component

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Math magicians 🔢 : Components

In this PR I :
* Add a components directory
* Add Calculator.js file in components dir
* Add component of the calculator
* Add Calculator.css to style calculator components
* Edit ReadMe file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,34 @@
...
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## 🤝 Contributing
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Contributions, issues, and feature requests are welcome!
```
Comment (zieeco) [inline README.md:39]: - Please, kindly add the link to the issues page of this repo

File: README.md [hunk 2]
```diff
@@ -1,70 +1,34 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+- MicroverseInc
```
Comment (zieeco) [inline README.md:48]: - Kindly add a license to this project

File: README.md [hunk 3]
```diff
@@ -1,70 +1,34 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
```
Comment (zieeco) [inline README.md]: - Kindly change this to the name of your project

File: README.md [hunk 4]
```diff
@@ -1,70 +1,34 @@
...
 
-## Available Scripts
+> Math Magicians
```
Comment (zieeco) [inline README.md]: - Please kindly some description about your project to give users an overview of what the project is all about

File: README.md [hunk 5]
```diff
@@ -1,70 +1,34 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (zieeco) [inline README.md:2]: - Kindly the getting started header and add some instruction on how users can have your project up and running locally in their machine.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ChrispaixK ,


Great job so far 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @ChrispaixK 👏🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**